docs/tctl/admin/cluster/index.md

```diff
@@ -0,0 +1,25 @@
+---
+id: index
```

Comment:
I am thinking we may be able to use the guide gen tool to generate these index pages.